helm/etcd-cluster-migrator/templates/job-psp-clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
This is `v1` in the `aws-operator`. Any reason to go with `v1beta1`?

Reply:
v1 is better, I copypasted from kiam-agent where for some reason we have the v1beta1

I will fix